com.huma.browser.desktop

```diff
@@ -0,0 +1,15 @@
+[Desktop Entry]
+Version=100.0a15
```

Comment:
definitely an invalid value. This is not the version of the application. This is the version of the desktop file specification followed by that file.